ex1/ex1.c

```diff
@@ -8,7 +8,17 @@
 
 int main(void)
 {
-    // Your code here
+    int x = 10;
+    pid_t pid = fork();
 
```

Comment:
It's always a good idea to put in some kind of error handler in case something goes wrong. I think the readme contains an example of this - if the fork fails, it allows the program to exit.